example/functions/create-session.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * This function creates a Stripe Checkout session and returns the session ID
+ * for use with Stripe.js (specifically the redirectToCheckout method).
+ *
+ * @see https://stripe.com/docs/payments/checkout/one-time
+ */
+
+const stripe = require('stripe')(process.env.REACT_APP_STRIPE_API_SECRET);
+
+/*
+ * Product data can be loaded from anywhere. In this case, we’re loading it from
+ * a local JSON file, but this could also come from an async call to your
+ * inventory management service, a database query, or some other API call.
+ *
+ * The important thing is that the product info is loaded from somewhere trusted
+ * so you know the pricing information is accurate.
+ */
+const inventory = require('./data/products.json');
+// const { validateCartItems } = useStripeCart;
+
+const validateCartItems = (inventorySrc, cartItems) => {
+  const validatedItems = Object.keys(cartItems).map((item) => {
+    const product = cartItems[item];
+    const validatedItem = inventorySrc.find(
+      (p) => p.name === cartItems[item].name
+    );
```

Comment:
We should have some error handling here - both for an empty `inventorySrc` and for not finding the item.

Reply:
I actually don't think we should have any error handling for whether an item is found or not, only because of the idea that the data coming from cartItem could be manipulated client side. I'd rather it skip something that can't be validated. If it wasn't intentional (some how) the user would have to go back and add whatever back to the cart.

Hope this makes sense